package/MDAnalysis/analysis/helanal.py

```diff
@@ -116,9 +123,6 @@
 .. autofunction:: helanal_main
 
 """
-from __future__ import print_function, division, absolute_import
```

Comment:
If this is targetting 1.0.1 then these need to come back in right?

Reply:
yes